cmd/build/helmify/static/README.md

```diff
@@ -163,6 +163,7 @@ information._
 | mutatingWebhookMatchConditions                             | The match conditions written in CEL to further refine which resources will be selected by the webhook. All match conditions must evaluate to true for the webhook to be called                                                                                                                 | `[]`                                                                                                                                                                  |
 | mutatingWebhookTimeoutSeconds                              | The timeout for the mutating webhook in seconds                                                                                                                                                                                                                                                | `3`                                                                                                                                                                   |
 | mutatingWebhookCustomRules                                 | Custom rules for selecting which API resources trigger the webhook. NOTE: If you change this, ensure all your constraints are still being enforced.                                                                                                                                            | `{}`                                                                                                                                                                  |
+| mutatingWebhookSubResources                                        | Rule for selecting which API subresources trigger the webhook.                                                                                                                                                                                             | `['pods/ephemeralcontainers', 'pods/exec', 'pods/log', 'pods/eviction', 'pods/portforward', 'pods/proxy', 'pods/attach', 'pods/binding', 'deployments/scale', 'replicasets/scale', 'statefulsets/scale', 'replicationcontrollers/scale', 'services/proxy', 'nodes/proxy', 'services/status']`                                                                                                                           |
```

Comment:
if we are moving these out as variables for mutating, we should do it for validating too?

Reply:
I thought about that, but didn't add to not introduce new helm variable unless we get a request for it. We are adding for mutating because of easy override. But I do see advantage of adding it there so that users can remove them without having to update chart. I will add it.

Reply:
Added to validating webhook as well. PTAL.